packages/jest-preset/transforms/babel.js

```diff
@@ -0,0 +1,57 @@
+const babel = require('@babel/core');
```

Comment:
this is an alternative to babel-jest that is:

- simpler,
- doesn't also try and support babel 6
- handles typescript out of the box if needed